js/main.js

```diff
@@ -1298,6 +1297,7 @@ function loadGameData() {
     //Enable/disable modules
     for (let id in modules) {
         const module = modules[id];
+        module.init();
         modules[id].setEnabled(gameData.currentModules.hasOwnProperty(module.name));
     }

```

Comment:
Der ganze Code Block ist hier unten etwas merkwürdig, weil der mit dem Laden der Daten nur so bedingt was zu tun hat. Das ist halt schon eher das initialisieren der Modules, egal ob nun Daten geladen wurden oder nicht. Würde ich in initModules() verschieben und einfach nach `loadGameData()` aufrufen.